### ISSUE TITLE
jhipster --blueprints react-native fails with ERROR! Package subpath './generators/app' 

### ISSUE THREAD

**Issue by BhaskarN10** [jhipster/generator-jhipster-react-native#1037]
on MS Windows Jhipster blueprint option fails (node v18.13.0 and npm 9.3.0)

>  jhipster --blueprints react-native
INFO! Using bundled JHipster
INFO! No custom sharedOptions found within blueprint: generator-jhipster-react-native at C:/Users/xxx/AppData/Roaming/npm/node_modules/generator-jhipster-react-native
INFO! No custom commands found within blueprint: generator-jhipster-react-native at C:/Users/BhaskarN/AppData/Roaming/npm/node_modules/generator-jhipster-react-native
ERROR! Package subpath './generators/app' is not defined by "exports" in C:\Users\BhaskarN\AppData\Roaming\npm\node_modules\generator-jhipster-react-native\node_modules\generator-jhipster\package.json
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './generators/app' is not defined by "exports" in C:\Users\xxx\AppData\Roaming\npm\node_modules\generator-jhipster-react-native\node_modules\generator-jhipster\package.json
    at new NodeError (node:internal/errors:400:5)
    at exportsNotFound (node:internal/modules/esm/resolve:361:10)
    at packageExportsResolve (node:internal/modules/esm/resolve:697:9)
    at resolveExports (node:internal/modules/cjs/loader:538:36)
    at Module._findPath (node:internal/modules/cjs/loader:607:31)
    at Module._resolveFilename (node:internal/modules/cjs/loader:1025:27)
    at Module._load (node:internal/modules/cjs/loader:885:27)
    at Module.require (node:internal/modules/cjs/loader:1105:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (C:\Users\xxx\AppData\Roaming\npm\node_modules\generator-jhipster-react-native\generators\app\index.js:5:22) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}

**Instructions To reproduce**

1)  npm install -g generator-jhipster-react-native

2) jhipster --blueprints react-native (fails to execute) 

- JHipster React Native Version:
- `jhipster info` output:

**Comment by mraible** [comment]
Can you try Node 16?On Jan 16, 2023, at 10:55, Bhaskarn10 ***@***.***> wrote:﻿
on MS Windows Jhipster blueprint option fails (node v18.13.0 and npm 9.3.0)

jhipster --blueprints react-native
INFO! Using bundled JHipster
INFO! No custom sharedOptions found within blueprint: generator-jhipster-react-native at C:/Users/xxx/AppData/Roaming/npm/node_modules/generator-jhipster-react-native
INFO! No custom commands found within blueprint: generator-jhipster-react-native at C:/Users/BhaskarN/AppData/Roaming/npm/node_modules/generator-jhipster-react-native
ERROR! Package subpath './generators/app' is not defined by "exports" in C:\Users\BhaskarN\AppData\Roaming\npm\node_modules\generator-jhipster-react-native\node_modules\generator-jhipster\package.json
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './generators/app' is not defined by "exports" in C:\Users\xxx\AppData\Roaming\npm\node_modules\generator-jhipster-react-native\node_modules\generator-jhipster\package.json
at new NodeError (node:internal/errors:400:5)
at exportsNotFound (node:internal/modules/esm/resolve:361:10)
at packageExportsResolve (node:internal/modules/esm/resolve:697:9)
at resolveExports (node:internal/modules/cjs/loader:538:36)
at Module._findPath (node:internal/modules/cjs/loader:607:31)
at Module._resolveFilename (node:internal/modules/cjs/loader:1025:27)
at Module._load (node:internal/modules/cjs/loader:885:27)
at Module.require (node:internal/modules/cjs/loader:1105:19)
at require (node:internal/modules/cjs/helpers:103:18)
at Object. (C:\Users\xxx\AppData\Roaming\npm\node_modules\generator-jhipster-react-native\generators\app\index.js:5:22) {
code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}

Instructions To reproduce


npm install -g generator-jhipster-react-native


jhipster --blueprints react-native (fails to execute)



JHipster React Native Version:
jhipster info output:

**Comment by BhaskarN10** [comment]
Good'ay Matt,

Issue resolved. I changed to node 16 and am able to execute the jhipster --blueprints react-native Successfully. 

Note: I first uninstalled node 18 that I installed.
 and then Installed node 16. 
 
Thanks for your prompt support. 

I take this opportunity to appreciate your contribution to Jhipster - which saved ton of time and effort in building web applications. 

Best Regards
Bhaskar / Atlanta GA.    Jan 2023.
